tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -600,6 +603,9 @@ def test_related_field_has_valid_related_name(self):
             '_+',
             '+',
         ]
+        if six.PY3:
+            related_names.append("試")
```

Comment:
do the two different identifiers test anything different? If so, I'd go with `related_names.extend([])`